wezterm-gui/src/overlay/copy.rs

```diff
@@ -1580,22 +1580,34 @@ pub fn copy_key_table() -> KeyTable {
         (
             WKeyCode::Char('c'),
             Modifiers::CTRL,
-            KeyAssignment::CopyMode(CopyModeAssignment::Close),
+            KeyAssignment::Multiple(vec![
+                KeyAssignment::CopyMode(CopyModeAssignment::Close),
+                KeyAssignment::CopyMode(CopyModeAssignment::ResetViewport),
```

Comment:
Maybe these should actually be the other way around, to preserve the old behaviour? Not sure.

Reply:
Swapped them around.